src/App.jsx

```diff
@@ -141,7 +141,8 @@ function App() {
         setReject(true);
       }
     }
-  }, [setLoggedIn, query]);
+  // eslint-disable-next-line 
+  }, []);
```

Comment:
why remove the dependencies? were they not needed? or is it that we just want this to run once only no matter what?

Reply:
The dependencies made the app render twice at startup, and the pop up shows up twice. The dependencies should not be needed since it's always on the `App` page anyways. I tested with the query in firebase and the env participant ID and study ID and it works the same.